test/safe/instance.test.js

```diff
@@ -0,0 +1,55 @@
+let Thing, SuperThing, fakeInstance
```

Comment:
Pared down version of `constructor.test.js` with assertions I felt were safe to make